### ISSUE TITLE
UI scales poorly on my iPhone 11 Pro

### ISSUE THREAD

**Issue by nsmaciej** [JeeZeh/React-Irish-Rail#29]
![A71F6999-F74B-4A1A-AF48-85EB15C60864_1_102_o](https://github.com/JeeZeh/react-irish-rail/assets/2553960/73588b36-bb84-4791-b6cb-c7c63f409021)

**Comment by jesse-ashmore** [comment]
Thanks for the issue!

Is the behaviour the same across multiple browsers on the device? @maciej-irl

**Comment by nsmaciej** [comment]
Yes, the screenshot above was in Arc (Safari with extra chrome), but it also happens in Safari (Safari).
